src/utils/self_test/SConscript

```diff
@@ -34,10 +34,10 @@ def scons():
 
     tenv = env.Clone()
 
-    tenv.AppendUnique(CPPPATH=['#/src/cart/src/cart', '#/src/cart/src/test'])
+    tenv.AppendUnique(CPPPATH=['#/src/cart', '#/src/tests/ftest/cart'])
```

Comment:
probably ok but is ftest where we want C source for the tests ?

Reply:
Where would it be a better place for them Jeff?
I placed them there since that was the original requirement in DAOS-4579:

> Description
> move src/cart/src/cart to src/cart:
> move src/cart/src/include/cart to src/include/cart
> move src/cart/src/swim to src/cart/swim
> move src/cart/src/self_test to src/utils
> move src/cart/src/crt_launch to src/utils
> move src/cart/src/ctl to src/utils
> move src/cart/src/utest to src/cart/test
> **move src/cart/src/test to src/tests/ftest/cart**

Reply:
Looks like all tests in src/cart/test are python/avocado tests, more suitable to move to src/tests/ftest/cart.
While src/cart/src/test are all C tests, doesn't seem fit in ftest. Do we have a common dir for all C tests other than src/tests/suite which is for daos_test only?

Reply:
In the case of Gurt, the test_gurt.c file is placed under /src/gurt/tests/ directory. I'm not really sure how to handle this. Any ideas?